CoreScriptsRoot/Modules/VR/NotificationHub.lua

```diff
@@ -655,9 +688,13 @@ do
 
 	local SendNotificationInfoEvent = RobloxGui:WaitForChild("SendNotificationInfo")
 	SendNotificationInfoEvent.Event:connect(function(notificationInfo)
-		local group = notificationsGroups[notificationInfo.GroupName or "Developer"]
+		local group = notificationsGroups[notificationInfo.GroupName or false] --avoid error by nil index
```

Comment:
This line seems like it could be more elegantly written:
`local group = notificationInfo.GroupName and notificationsGroups[notificationInfo.GroupName]`